Sources/FluentUI_iOS/Components/Notification/FluentNotification.swift

```diff
@@ -101,6 +108,8 @@ public struct FluentNotification: View, TokenizedControlView {
                 actionButtonTitle: String? = nil,
                 actionButtonAction: (() -> Void)? = nil,
                 showDefaultDismissActionButton: Bool? = nil,
+                showActionButtonAndDismissButton: Bool = false,
```

Comment:
No need to act on this PR, but noting that this class's `init` is getting a little out of hand in terms of arguments. Probably time to circle back on the idea of using a configuration struct of some sort, though we need to work out the details on that still.